actors/migration/top.go

```diff
@@ -19,30 +22,66 @@ import (
 	"github.com/filecoin-project/specs-actors/v2/actors/states"
 )
 
+var (
+	defaultMaxWorkers = 16
+)
+
 // Config parameterizes a state tree migration
 type Config struct {
+	MaxWorkers int
 }
 
 func DefaultConfig() Config {
-	return Config{}
+	return Config{
+		MaxWorkers: defaultMaxWorkers,
+	}
 }
 
 type PowerUpdates struct {
 	claims map[address.Address]power0.Claim
 	crons  map[abi.ChainEpoch][]power0.CronEvent
 }
 
+type PowerUpdater interface {
+	Apply(*PowerUpdates)
+}
+
+type claimUpdate struct {
+	addr  address.Address
+	claim power0.Claim
+}
+
+// Overwrite claim
+func (c claimUpdate) Apply(p *PowerUpdates) {
```

Comment:
This is an unusual pattern in go (although I see why you're using it). I'd at least call it `x.ApplyTo(y)`, usually `x.Apply(y)` applies `y` to `x`.